### PR TITLE
Fix processing of stdin with --stdin-display-name

### PR DESCRIPTION
**Pull request by blueyed**
Fixes https://github.com/gforcada/flake8-isort/issues/47.

Relevant: http://flake8.pycqa.org/en/latest/plugin-development/plugin-parameters.html#indicating-desired-data

### CODE REVIEW

**Comment by gforcada** [comment]
@blueyed thanks for reporting the issue and taking the extra time to actually fix it! :bowing_man: would you mind to fixup the commit and add a change log entry on `CHANGES.rst`?

File: flake8_isort.py [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8 -*-
 from difflib import Differ
-from flake8_polyfill import stdin
```
Comment (gforcada) [inline flake8_isort.py]: If this is not needed anymore, we could remove it from setup.py. Would you mind doing it? TIA

**Comment by gforcada** [comment]
@blueyed one last comment: which flake8 versions are we expected to support with these changes? :thinking: I thought that the polyfill was meant to retain compatibility with flake8 2.x I'm ok with dropping support for it, but we should state it clearly somewhere.

**Comment by blueyed** [comment]
Thanks for the feedback/review.
I have not checked what flake8 versions this requires, but we should figure that out.
Then Travis could also run an older (but supported) version for example.

**Comment by blueyed** [comment]
`install_requires` has flake8 >= 3.2.1 already.

**Comment by blueyed** [comment]
Pushed some fixups.
You can squash-merge them from the GitHub UI.

**Comment by gforcada** [comment]
@blueyed thanks!

**Comment by gforcada** [comment]
@blueyed flake8-isort 2.4 released! :tada:

**Pull request merged**